src/Microsoft.AspNetCore.BuildTools.ApiCheck/ApiListingComparer.cs

```diff
@@ -76,6 +76,24 @@ private void CompareMembers(TypeDescriptor type, TypeDescriptor newType, List<Ap
                     }
                 }
             }
+
+            if (type.Kind == TypeKind.Interface && type.Members.Count < newType.Members.Count)
+            {
+                var members = newType.Members.ToList();
```

Comment:
FilterExclusions also modifies the exclusions list, so I don't want to do it in a query style